docs/src/modules/java/pages/author-your-first-service.adoc

```diff
@@ -182,10 +186,119 @@ include::example$doc-snippets/src/main/java/com/example/api/ExampleEndpoint.java
 ----
 <1> Returning a record that gets serialized as JSON
 
+Restart the service and curl this command:
+
 [source, command line]
 ----
 curl localhost:9000/hello/hello/Bob/30
 ----
 
+== Explore the local console
+
+The Akka local console is a web-based tool that provides a convenient way to view and interact with your running service.
+
+=== Install the Akka CLI
+
+The local console requires that you install the Akka CLI first.
+
+Install the Akka CLI:
```

Comment:
Do we have to inline the installation instructions, or could we link to it?
```
xref:reference:cli/installation.adoc[install the Akka CLI]
```

Reply:
@ennru [suggested we make a special exception and inline installation instructions on this page](https://github.com/lightbend/kalix/issues/12159#issuecomment-2509081943). I agree with this idea. The intent is to prevent a new user from needing to swiitch to a different doc page during the getting started experience.

Reply:
ok